src/ctap/mod.rs

```diff
@@ -778,8 +778,11 @@ impl CtapState {
             enterprise_attestation,
         } = make_credential_params;
 
-        // TODO: Get the slot id from active token state when multi-PIN is enabled.
-        let slot_id = Some(0);
+        let slot_id = if storage::has_multi_pin(env)? {
+            self.client_pin.get_slot_id_in_use()
+        } else {
+            Some(0)
+        };
```

Comment:
Looks like this pattern could be a helper function in ClientPin, e.g. get_slot_id_in_use_or_default(env) which returns the current slot id if multipin otherwise 0.

Reply:
done